lib/alpha_taxonomy/taxon_link_deleter.rb

```diff
@@ -0,0 +1,27 @@
+module AlphaTaxonomy
+  class TaxonLinkDeleter
+    def initialize(logger: Logger.new(STDOUT))
+      @log = logger
+    end
+
+    def run!
+      base_paths = ENV.fetch("TAXON_CLEANUP").split(",")
```

Comment:
Yeah, this whole ENV thing is definitely becoming a bit leaky. I'd decided to defer this change but it makes sense given the rake tasks are now the main entry-point to all this logic. Will update.